source/codegen/metadata/nidcpower/nidcpower.proto

```diff
@@ -41,8 +42,11 @@ service NiDCPower {
   rpc ConfigureDigitalEdgeSourceTriggerWithChannels(ConfigureDigitalEdgeSourceTriggerWithChannelsRequest) returns (ConfigureDigitalEdgeSourceTriggerWithChannelsResponse);
   rpc ConfigureDigitalEdgeStartTrigger(ConfigureDigitalEdgeStartTriggerRequest) returns (ConfigureDigitalEdgeStartTriggerResponse);
   rpc ConfigureDigitalEdgeStartTriggerWithChannels(ConfigureDigitalEdgeStartTriggerWithChannelsRequest) returns (ConfigureDigitalEdgeStartTriggerWithChannelsResponse);
+  rpc ConfigureLCRCompensation(ConfigureLCRCompensationRequest) returns (ConfigureLCRCompensationResponse);
```

Comment:
Regarding the new methods exposed here, someone from MI should verify these are approved/OK to include here. @marcoskirsch @ni-jfitzger thoughts?

Reply:
Talked with Dan offline about this and he saw that these methods are released and also they saw these get included with the hapigen changes in AzDo so this should be fine. Marking resolved.